KitePrintSDK/src/main/java/ly/kite/print/SubmitPrintOrderRequest.java

```diff
@@ -14,12 +19,12 @@ public SubmitPrintOrderRequest(PrintOrder printOrder) {
         this.printOrder = printOrder;
     }
 
-    public void submitForPrinting(final SubmitPrintOrderRequestListener listener) {
+    public void submitForPrinting(Context context, final SubmitPrintOrderRequestListener listener) {
         assert req == null : "you can only submit a request once";
 
         JSONObject json = printOrder.getJSONRepresentation();
-        String url = String.format("%s/print", KitePrintSDK.getEnvironment().getPrintAPIEndpoint());
-        req = new BaseRequest(BaseRequest.HttpMethod.POST, url, null, json.toString());
+        String url = String.format("%s/print", KiteSDK.getInstance( context ).getPrintAPIEndpoint());
```

Comment:
`getPrintAPIEndpoint` probably one for the renaming todo's in the future as we're less and less about traditional print now